lib/iex/lib/iex/introspection.ex

```diff
@@ -141,6 +146,20 @@ defmodule IEx.Introspection do
     end
   end
 
+  defp callback_module(mod, fun, arity) do
+    mod.module_info(:attributes)
+    |> Stream.filter(&match?({:behaviour, _}, &1))
```

Comment:
These two lines (current +1 below) could be simplified with single `Stream.filter_map/3` call.